Source/WebKit/UIProcess/mac/WKTextAnimationManager.mm

```diff
@@ -95,19 +96,30 @@ - (void)addTextAnimationForAnimationID:(NSUUID *)uuid withData:(const WebKit::Te
         break;
     case WebKit::TextAnimationType::Source:
         effect = adoptNS([PAL::alloc_WTReplaceSourceTextEffectInstance() initWithChunk:chunk.get() effectView:_effectView.get()]);
+        if ([effect respondsToSelector:@selector(setPreCompletion:)]) {
```

Comment:
Assuming this respondsToSelector was originally adding for staging purposes, it can probably be removed now, right?